docs/tequila-api.md

```diff
@@ -3,6 +3,27 @@ title: What is Tequila API?
 sidebar_label: What is Tequila API?
 ---
 
-_This page is a stub._
+With the [Tequila](https://tequila.kiwi.com) platform, you can build your o travel business application for free, with the [kiwi.com](https://www.kiwi.com)'s content, technology, and services.
 
-Tequila API is an API.
+All you need to start working with the **Tequila** API is you need to be registered and have your API key.
+Check the Guide: [Get your Tequila API key](guide-tequila-api-key.md) for information on how to grab your API Key.
+
+To implement the basic search, you need the `/locations` and `/search` API calls.
+
+- **Locations API**: search for the cities, airports, stops, and other locations.([docs](https://tequila.kiwi.com/portal/resources/locations_api))
+
+- **Search API**: this API allows you search for one-way or return type of itineraries.([docs](https://tequila.kiwi.com/portal/resources/search_api))
+
+- **Mutlicity API**: this API provides the multicity search. The multicity trip is a journey when the passenger travels from one city to another, stays there for a couple of days and then continues the journey to another city. ([docs](https://tequila.kiwi.com/portal/resources/multicity_api))
```

Comment:
`a journey where the passenger...`

Reply:
I am not sure if this is better, so must ask for another opinion.
@RobinCsl what do you think?  `a journey where the passenger...` or `a journey when the passenger...`

Reply:
@FilipMessa I think the correct way is `A journey where + something`

Reply:
OK, thanks I will change as you suggest

Reply:
Perfect! I think it's all good, only one small thing, there's a typo in `Mutlicity API`, with that I think it's good to go 💃